src/session_impl.cpp

```diff
@@ -5223,8 +5218,13 @@ namespace {
 		}
 
 		std::vector<listen_interface_t> new_ifaces;
+		for (auto& ls : current_ifaces)
+		{
+			ls.ssl = true;
+		}
```

Comment:
oh. good catch. when I first looked at the ``transform`` below I thought this is what it did, but I forgot to remove this